cfglib.py

```diff
@@ -105,6 +105,10 @@
                help='Size of one chunk to transfer via SSH'),
     cfg.StrOpt('group_file_path',
                help='Path to file with the groups of VMs'),
+    cfg.BoolOpt('all_vms', default=False,
+                help="Migrate all VM's from all tenants. User, specified in "
```

Comment:
This says "Migrate all VMs from all tenants", do we ever check if user specified in config has admin privileges to migrate all VMs? Or will we migrate some subset of VMs user has access to, and then get error?